DEPS

```diff
@@ -0,0 +1,85 @@
+use_relative_paths = True
+
+vars = {
```

Comment:
Hm. This isn't the DEPS file format I remember.  Then again, it's been a long time and my memory could be considered....fuzzy.